projects/Mallard/ios/Podfile.lock

```diff
@@ -343,4 +342,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 8f0bbc01628753467e2078e01bc0c2b5099afb92
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.7.5
```

Comment:
Can you upgrade your local cocoapod version to avoid this?

    sudo gem install cocoapods -v 1.8.4